stripe/stripe-android
# Add allowCreditCards field

# Summary


- Add `allowCreditCards` field to `GooglePayJsonFactory`
- Add field to toggle `allowCreditCards` in `GooglePayLauncher`
    - Set to a non-null value to pass it to GooglePay
- Added toggle to `GooglePayLauncherPlaygroundActivity`

To test this change: 
- Uncomment `GooglePayLauncherPlaygroundActivity` in `LauncherActivity`
- Make sure you have some credit cards and non credit cards in your Google Pay Wallet
- Launch Stripe Payments Example app -> Test GooglePayLauncher Configurations
- Toggle the radio button to false
- Press Pay with Google
- Assert that credit cards are not in the list of payment methods in your wallet

# Motivation


Fixes https://github.com/stripe/stripe-android/issues/5016

# Testing


# Changelog


- [Added] Add `allowCreditCards` to `GooglePayLauncher` and `GooglePayPaymentMethodLauncher`
